iothub_client/tests/common_e2e/iothubclient_common_e2e.c

```diff
@@ -104,6 +114,14 @@ typedef struct EXPECTED_SEND_DATA_TAG
     IOTHUB_MESSAGE_HANDLE msgHandle;
 } EXPECTED_SEND_DATA;
 
+#define MAX_SEND_DATA_BUNDLE_ITEM_COUNT 10
```

Comment:
Not sure how `MAX_SEND_DATA_BUNDLE_ITEM_COUNT` is tied to other places in program other than defining memory size of `items` right below.  Is this what was supposed to be 5, since it represest max count outstanding?